### PR TITLE
mask sliders for size/feather use log scale

### LINKED ISSUE #14079: [FR] An efficient method for making large changes to brush size
**Is your feature request related to a problem? Please describe.**

Recently I've been using the brush to paint masks more often, and especially in more cases where the brush size needs to be changed over a large range. There doesn't seem to be an efficient way to do this, necessitating a ridiculous amount of scroll-wheel actuation.

**Describe the solution you'd like**

For my purposes, it would be enough to be able to create a shortcut which directly changed the size in larger increments than the scroll-wheel. (I thought this might be possible already, but doesn't seem to be, unless I'm missing something). A "modifier" shortcut which increased the increment of the scroll-wheel itself would also solve the problem.

**Alternatives**

The most complete solution would be to add a slider for brush size to the drawn mask control set, which could then be accessed in all the currently available ways, but I can understand that adding another slider to an already busy set might not please everyone.

**Additional context**

IIRC before the shortcut system overhaul, when there were default shortcuts for brush size (square bracket keys), holding those keys would continuously change the size until released. Shortcuts created for this now don't have this behavior, but I'm not sure it was the best way to solve this problem, so I'm not specifically requesting that.

### PR DESCRIPTION
**Pull request by dterrahe**
fixes #14079

Also adds opacity slider to mask manager during the creation of new shapes and allows size/hardness slider shortcuts to adjust these properties while drawing.

And fixes a few minor bugs/inconsistencies.

Release Note:
- the mask shape size/feather/hardness sliders in the masks manager module now display in log scale and scrolling over them makes relative adjustments, just like (Shift) scrolling over the shape itself. Ctrl or Shift will make fine or coarse adjustments, also with shortcuts if fallbacks are enabled. Shortcuts assigned to the sliders can be used to adjust brush size/hardness while drawing.

### CODE REVIEW

**Comment by TurboGit** [comment]
I have been preparing this code refactoring yesterday : #14118
We have conflicts between this PR and mine.

**Comment by dterrahe** [comment]
That would mostly (only?) be the introduction of DT_REL_UP/DOWN which I can revert since your PR addresses the same (1.03 vs 1/0.97) "bugs".

**Comment by dterrahe** [comment]
Done

File: src/bauhaus/bauhaus.c [hunk 1]
```diff
@@ -1149,6 +1149,14 @@ static float _reverse_linear_curve(float value, dt_bauhaus_curve_t dir)
   return 1.0 - value;
 }
 
+float _curve_log10(float inval, dt_bauhaus_curve_t dir)
```
Comment (TurboGit) [inline src/bauhaus/bauhaus.c:1152]: **static** ... **const** float ... **const** dt_bauhaus_curve_t

**Review by TurboGit — CHANGES_REQUESTED** [review]
Minor style fix.

**Pull request merged**